File: java/Handphone.java
class Handphone extends Elektronik {
    private String ram;
    private String penyimpanan;

    public Handphone() {
        super();
        ram = "";
        penyimpanan = "";
    }

    public Handphone(String nama, String harga, String merek, String garansi, String daya, String warna, String ram, String penyimpanan) {
        super(nama, harga, merek, garansi, daya, warna);
        this.ram = ram;
        this.penyimpanan = penyimpanan;
    }

    public String getRam() {
        return ram;
    }

    public void setRam(String ram) {
        this.ram = ram;
    }

    public String getPenyimpanan() {
        return penyimpanan;
    }

    public void setPenyimpanan(String penyimpanan) {
        this.penyimpanan = penyimpanan;
    }
}

File: python/Handphone.py
from Elektronik import Elektronik

class Handphone(Elektronik):
    def __init__(self, nama = "", harga = "", merek = "", garansi = "", daya = "", warna = "", ram = "", penyimpanan = ""):
        super().__init__(nama, harga, merek, garansi, daya, warna)
        self.__ram = ram
        self.__penyimpanan = penyimpanan

    def getRam(self):
        return self.__ram

    def setRam(self, ram):
        self.__ram = ram

    def getPenyimpanan(self):
        return self.__penyimpanan

    def setPenyimpanan(self, penyimpanan):
        self.__penyimpanan = penyimpanan